tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -174,6 +174,10 @@ Spawn Notebook With Arguments  # robocop: disable
          Click Element  xpath://span[@id='jupyterhub-logo']
       ELSE
          Click Element  xpath://span[@id='jupyterhub-logo']
+         Sleep  ${retries_delay}
+      END
+      IF ${spawn_fail} == True
+         Fail  msg= Spawner failed loading after ${retries} retries
       END
```

Comment:
This whole block should go after the last END (right below, line 182)

Reply:
good catch, fixed